pkg/vsphere/extraconfig/store.go

```diff
@@ -43,7 +42,7 @@ func (t *MapStore) Get(key string) (string, error) {
 
 	val, ok := t.store[key]
 	if !ok {
-		return "", os.ErrNotExist
+		return "", ErrKeyNotFound
```

Comment:
I made an attempt to find all callers and fix them. Don't think I missed any in our source at least.